group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -131,6 +131,11 @@
  */
 public class GroupMetadataManager {
 
+    /**
+     * A member epoch of <code>-1</code> means that the member wants to leave the group.
+     */
+    public static final int LEAVE_GROUP_EPOCH = -1;
```

Comment:
I wonder if we should rather define this on in `ConsumerGroupHeartbeatRequest`. What do you think?

Reply:
Makes sense.

Reply:
Done.